js/basket-signup.js

```diff
@@ -0,0 +1,78 @@
+// Posting to Basket
```

Comment:
Copied this file from Foundation repo with some modification (see the two comments below)